### ISSUE TITLE
design: Prototype the login, sign up, user's profile and reset password pages

### ISSUE THREAD

**Issue by junocchi** [Pensano-dev/aura-mobile-app#1]
Please consider the pages 1-4 https://www.canva.com/design/DAF8hRD8_Qk/kyDoO2jsyyOxz74k0SdB5g/view?mode=prototype.

**Comment by pablisch** [comment]
Hey Ju, these look great and give us a great place to build from! Thanks for doing this! A few points for discussion:

GENERAL: I think there might be an accessibility issue with the colour palette having blue on blue and possible low contrast issues from what I experienced on the QE course. @ChalkandFeather will be able to give proper guidance on this and it is not anything we need to sort right away. Some elements are a little small for me personally, but again these are things we can easily discuss and irion out.

PAGE 1: From discussions I expected to have Google auth as the primary option with 'secondary' option of email signup. I think this is a good way forward for a mobile app. We could have the page kind of assuming Google login and opening a modal for email signup to give a really clean page.
Personally, when using a non-Google login, I would prefer username rather than email login as it is much easier for users. I don't feel strongly about this but am interested in arguments against it.

As a design template this is great. I'd love to see a template for the main page, including how we offer signup and login. I always imagined this as a free-use app first of all, by which I don't mean anything about money but rather as much as possible, the app does not rely on being logged in at all and is free to use straight from the main page without any need to login. In this case, we should consider what functionality requires a login, e.g. leaving reviews/ratings, saving places (non-MVP), etc.

**Comment by pablisch** [comment]
Agreed that profile functionality will be minimal at first and added to as we go. I take the design as a placeholder for what will come in time but I like the premise. 

I think one thing I would like to see design-wise is how the elements are accessed, e.g. login, signup, profile/settings, return to homepage/attribute-selector. I find it hard to visualize with a profound lack of mobile experience. A menu-bar at the top? At the bottom? A hamburger menu? It would be nice to start with a structural design decision to move forward with. Functionally, I think I favour a bottom menu-bar.

![Image](https://github.com/Pensano-dev/aura-mobile-app/assets/19639889/86856a3d-2614-4065-819e-97b3cdfe54da)

**Comment by pablisch** [comment]
NOTE: This ticket altered from original remit of login, signup, profile pages, etc.
